rcl/test/rcl/common.hpp

```diff
@@ -0,0 +1,56 @@
+// Copyright 2020 Open Source Robotics Foundation, Inc.
```

Comment:
maybe the name of the file could be `wait_for_entity_helpers`, or sth like that

Reply:
OKay